development/editor/introduction_to_editor_development.rst

```diff
@@ -0,0 +1,75 @@
+.. _doc_introduction_to_editor_development:
+
+Introduction to editor development
+==================================
+
+On this page, you will learn:
+
+- The **design decisions** behind the Godot editor.
+- How to work efficiently on the Godot editor's C++ code.
+
+This guide is aimed at current or future engine contributors.
+To create editor plugins in GDScript, see :ref:`doc_making_plugins` instead.
+
+.. seealso::
+
+    If you are new to Godot, it's recommended to read
+    :ref:`doc_godot_design_philosophy` before continuing. Since the Godot editor
```

Comment:
```suggestion
    If you are new to Godot, we recommended you to read
    :ref:`doc_godot_design_philosophy` before continuing. Since the Godot editor
```